infrastructure/builder/src/generate/generators/docs-tocs.js

```diff
@@ -152,14 +152,19 @@ exports.tasks = [{
   requires: [],
   provides: ['docs-toc'],
   run: async (requirements, utils) => {
-    const files = await mdParseDir(DOCS_DIR, { dirnames: true });
+    const filesWithExtensions = await mdParseDir(DOCS_DIR, { dirnames: true });
+    // strip .md and .mdx extensions frmo those filenames..
+    const files = Object.assign(
+      ...Object.entries(filesWithExtensions)
+        .map(([filename, value]) => ({[filename.replace(/\.mdx?/, '')]: value})));
+    console.log(Object.keys(files));
```

Comment:
`console.log` ?

Reply:
Oh, thanks, I had deleted but not committed the change.

@helfi92 can you double-check me in thinking that this is unused in the UI?